PKGBUILD

```diff
@@ -1,67 +1,66 @@
-# Derived from PKGBUILD for Inox
-# Original Inox PKGBUILD authors:
-# Maintainer: Michael Egger <gcarq@archlinux.info>
-# Contributor: Evangelos Foutras <evangelos@foutrelis.com>
+# Maintainer: Ungoogled Software Contributors
+
+# Based on aur/chromium-vaapi, with ungoogled-chromium patches
+
+# Maintainer: Evangelos Foutras <evangelos@foutrelis.com>
 # Contributor: Pierre Schmitz <pierre@archlinux.de>
 # Contributor: Jan "heftig" Steffens <jan.steffens@gmail.com>
 # Contributor: Daniel J Griffiths <ghost1227@archlinux.us>
 
 pkgname=ungoogled-chromium
-# Commit or tag for the upstream ungoogled-chromium repo
-_ungoogled_version='81.0.4044.138-1'
-_chromium_version=${_ungoogled_version%-*}
-_ungoogled_revision=${_ungoogled_version#*-}
-pkgver=${_chromium_version}
-pkgrel=$_ungoogled_revision
+pkgver=83.0.4103.61
+pkgrel=1
+_pkgname=ungoogled-chromium
+_ungoogled_ver=83.0.4103.61-1
```

Comment:
Maybe it would be more intuitive to write this as `_ungoogled_ver=${pkgver}-1`?

Reply:
It would be, but I have it that way since sometimes a new update is out and it is just a security fix, and most of the time an old patchset can be combined with a new chromium release. That way users won't be vulnerable until you accept a PR on the main repo.

Reply:
I think the better way to handle this is to add automation for security releases on the main repo, and I'm not convinced that adding hacks to fast track security fixes is a great idea. But if @wchen342 or @braewoods feel this is acceptable, I think you should at least document this use-case in a comment.

Reply:
Considering Arch is updated a lot more frequently than other distributions I think this is acceptable, but yes a note would be nice.

Reply:
alright, I'll add a comment to address that